modules/custom/az_core/src/Drush/Commands/AZEntityListCommands.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\az_core\Drush\Commands;
+
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drush\Attributes as CLI;
+use Drush\Commands\DrushCommands;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * A Drush commandfile.
+ */
+final class AZEntityListCommands extends DrushCommands {
+
+  /**
+   * The entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * Constructs a new AzEntityListDrushCommands object.
+   */
+  public function __construct(
+    EntityTypeManagerInterface $entity_type_manager
+  ) {
+    parent::__construct();
+    $this->entityTypeManager = $entity_type_manager;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('entity_type.manager'),
+    );
+  }
+
+  /**
+   * List entities enabled on an Arizona Quickstart site.
+   *
+   * @return \Consolidation\OutputFormatters\StructuredData\RowsOfFields
+   *   A table with a row for each asset type and count.
+   */
+  #[CLI\Command(name: 'az-entity-list:list', aliases: ['ael'])]
+  #[CLI\DefaultFields(fields: [
+    'entity_type',
+    'bundle',
+    'count',
+    'module',
+  ])]
+  #[CLI\FieldLabels(labels: [
+    'entity_type' => 'Entity type',
+    'bundle' => 'Bundle',
+    'count' => 'Count',
+    'module' => 'Entity Type Provider',
+  ])]
+  #[CLI\Usage(name: 'drush az-entity-list:list', description: "List entities enabled on an Arizona Quickstart site.")]
+
+    public function list(array $options = ['format' => 'table']): RowsOfFields {
+    $all_results = [];
+    $entity_types = array_keys($this->entityTypeManager->getDefinitions());
+
+    foreach ($entity_types as $entity_type) {
+      // If the entity has bundle types add them to the list. Otherwise, just
+      // use the entity type as the bundle.
+      $bundle_types = $this->getBundleTypes($entity_type);
+      $entity_provider = $this->entityTypeManager->getDefinition($entity_type)->getProvider();
+      foreach ($bundle_types as $bundle) {
+        $count = $this->getEntityCount($entity_type, $bundle);
+        if ($count === 0) {
+          continue;
+        }
+        $all_results[] = [
+          'entity_type' => $entity_type,
+          'bundle' => $bundle,
+          'count' => $count,
+          'module' => $entity_provider,
```

Comment:
I think this should change to 
```suggestion
          'entity_type_provider' => $entity_type_provider,
```
In order to make way for `$bundle_type_provider`